### ISSUE TITLE
Community support (chat, slack, forum, etc)

### ISSUE THREAD

**Issue by octoxalis** [11ty/eleventy#149]
After comparing Hugo and Eleventy I think the later really deserves a wider adoption by those in search of a fine static site generator as I'm.
However there's no forum or Stack Exchange community to find some help. It would be nice to create one, don't you think so?

**Comment by chrisdmacrae** [comment]
@zachleat spectrum.chat community?

**Comment by DirtyF** [comment]
update: ~~https://spectrum.chat/eleventy~~

**Comment by chrisdmacrae** [comment]
Awesome @DirtyF.

**Comment by chrisdmacrae** [comment]
@zachleat if you're alright with spectrum, I'll submit a PR to add an icon to the website, and @DirtyF can give you owner/admin of the community.

**Comment by zachleat** [comment]
Hey—I’m glad this exists but my schedule and burnout-fighting happiness level can’t sustain live chat support of the project. I would really rather issues get funneled through GitHub for search-ability and re-use—even if they’re just questions. The async nature of that—in my opinion—will be better for the long term success of the project, if only for the maintainer’s happiness level 😅

This, of course, will probably change if the project is successful and has new requirements of scale.

**Comment by zachleat** [comment]
What are the benefits of Stack Exchange over GitHub issues? Maybe there is some benefit I’m missing there.

**Comment by zachleat** [comment]
I’m going to close this for now but please funnel questions through GitHub issues for now!

**Comment by zachleat** [comment]
More requests:
https://twitter.com/miklb/status/1078847777042501632
https://twitter.com/csakiistvan/status/1092660053163868160
https://twitter.com/gerwitz/status/1082724060344344576
https://twitter.com/cfloth/status/1134868599045312513

We ran an old poll here:
https://twitter.com/eleven_ty/status/1092975339167510528

**Comment by edwardhorsford** [comment]
I'd *love* a slack channel.

**Comment by Ryuno-Ki** [comment]
Well, if you go for chat, publish the logs and make it indexable.

Other than that: Is there anything stopping the _community_ to rally around other forums?

I'd expect them to forward questions here by filling issues.

**Comment by kleinfreund** [comment]
> if you go for chat, publish the logs and make it indexable.

I don’t expect any chat platform to publish logs of my messages in any form. Content that should be accessible via search on a later point in time shouldn’t be published in the form of chat messages. In fact, I would prefer some of my chats to automatically delete messages older than some time span.

**Comment by Ryuno-Ki** [comment]
Well, coming from IRC I'm used to it.
I still see folks using it, e.g. [IndieWeb](https://chat.indieweb.org/2019-06-05#bottom).

Maybe the expectation changed at some point?

Without an index, chat is pretty much useless for knowledge gathering (like @zachleat quoted earlier).

**Comment by zachleat** [comment]
Reopening

**Comment by miklb** [comment]
The IndieWeb model of web/IRC/Slack integration might be the ideal, but keep in mind beyond full searchable logs, there is a heavy reliance on using the wiki too.

It is also a much broader topic.

All of that said, i would definitely idle in an eleventy channel. Answer a question when I can, ask one if I'm stuck.

**Comment by Ryuno-Ki** [comment]
Sure. But how to turn those answers into a way, that the next person can discover it, i.e. not has to ask in the first place?

**Comment by miklb** [comment]
That is an age old question that I haven't seen solved.

**Comment by Ryuno-Ki** [comment]
Hey @zachleat,
I've seen you're part of IndieWeb movement as well.
I wonder, whether [WebSub](https://indieweb.org/WebSub) would be helpful here (I got to know it as it was known as PubSubHubbub …).

Not that I am aware of any chat solution that would offer it … but at least, it would be crawlable if it were.

**Comment by AdamSzakal** [comment]
A place to ask dumb questions without having to worry about ones embarrassing lack of understanding publicly and permanently being saved would be truly helpful. Asking for friend.

**Comment by coolsoftwaretyler** [comment]
Gosh I'm of two minds on this. For one, I think the Eleventy GitHub issues is my favorite repo issue tracker *because it's being used for this kind of thing*. I think it's cool to see people using this software answering/asking questions right on the project repo. 

But I would also love some kind of community where I could more casually chat. I find myself editing and re-editing my correspondence here because it's so public and semi-permanent. In a slack channel, there's less cognitive overhead for me as a user to ask and contribute. 

But overall I think the priority needs to be the maintainer's energy level, so I can live with GitHub issues only if it means Eleventy will continue to grow. 

As for a middle-ground option, is there a way we can set up an informal community on Slack or some other chat platform (IRC, discord, etc.) that is unofficial and doesn't require maintainers to moderate/take ownership of it?

**Comment by TigersWay** [comment]
Anyone would like to create/share a "room" on Gitter?
We can try, and we'll see.

**Comment by chrissy-dev** [comment]
I'm with @zachleat - using GitHub issues keeps everything close to the core of the project and is easier to maintain and track for @zachleat and the other project collabrators. 

If we end up with a slack channel it _could_ turn into massive conversations with important solutions and issues missed.

That being said, I think building a positive, diverse and inclusive community around 11ty is a great idea. We would need community guidelines and a small team of people to maintain the current culture of the project.

Also I agree with @Ryuno-Ki, it needs to be indexed and searchable. 👍

**Comment by DirtyF** [comment]
From previous experience, having a public forum helps the core team focus on the issues and not the education part. It's a matter of signal/noise in issues, when it becomes not manageable, then maybe it will be time to move those conversations elsewhere.

**Comment by octoxalis** [comment]
@DirtyF  I definitely agree on this **separation of concern** between **education** and **issues**.

**BTW**, I've created a personal blog devoted to some educational purpose around **11ty**. It's called [11tyTips](https://11tytips.netlify.com) and is, at the moment, a _work in progress_, but it would be of great help if some of you could have a look for giving me advises. It's hosted on Netlify and it soon will have a domain of its own (probably **11tytips.io**, **if Zach agrees**, because I don't want to raise a confusion with **11ty.io**).

I also plan to create a [GitHub template repository](https://help.github.com/en/articles/creating-a-template-repository) for this site to help newbies quicky setup a fresh new site with goodies like CSS, JS, comments, etc. and begin to dive in the fascinating realm of **11ty** and JAMstack...
Suggestions are more than welcome.

**Comment by Ryuno-Ki** [comment]
Can we move the tips of the official documentation there?
What about other recipes?

**Comment by octoxalis** [comment]
Of course you may, it's the purpose of this blog!

Being a bit busy at the moment, just let me few days to finalize some contents.

I plan to write new recipes regularly. And any kind of contribution is most than welcome...

**Comment by Ryuno-Ki** [comment]
No, I mean removing the [quicktips](https://github.com/11ty/11ty.io/tree/59ec2dcf84ba933ff618b602b8619aaa1aa77bfc/docs/quicktips) from the docs and link to your page instead.
Something Zach would need to decide.

Once you're finished polishing a link to the site from 11ty's README could be handy.

**Comment by octoxalis** [comment]
It's a good suggestion and I adopt it.
